platform/resources/user-retirement-collector.sh

```diff
@@ -8,6 +8,10 @@ set -ex
 # just stores a filename in the environment (rather than the content).
 env
 
+# create and activate a virtualenv
+virtualenv --python=python3.8 venv
```

Comment:
Is there a chance this venv will already exist from a previous run? I'm not where this sits in the process, but I was under the impression that workers will be re-used more than once. Not sure if this will fail if `venv` already exists.

Reply:
whoops, I think I missed adding a `--clear` here

Reply:
I addressed this by both adding `--clear` (a safety measure) and also by creating a unique venv directory name for each build, as discussed on slack with @doctoryes .  The idea is that we don't need them to be unique today, but in the future somebody could flip the switch to make workers allow concurrent jobs and this would protect against that.  I'm on the fence between adding complexity vs. ignoring a possible time bomb so I just picked one at random (the former).